### ISSUE TITLE
Grade display inconsistent when using alternate grade with aid modification

### ISSUE THREAD

**Issue by braeuni** [theCrag/website#4052]
When grading a pitch with an additional aid grade like described here
![grafik](https://user-images.githubusercontent.com/7069305/165484489-a247780c-7865-4c22-90a8-a0de2195ce07.png)

the overall grade of the route is displayed incorrectly https://www.thecrag.com/de/klettern/austria/route/5441453259
Also the Aid grade is only displayed on grade citation, it would be nice if it would appear in a more ui friendly way.

![grafik](https://user-images.githubusercontent.com/7069305/165488119-62d58c83-e242-47f3-b309-07a37395d844.png)

**Comment by scd** [comment]
I believe I have fixed this. The problem was that the syntax for brackets assumed all the pitches were in the square brackets, so `2,4-,4,4+,2,7,3,3,3 [2,4-,4,4+,2,4+ A1,3,3,3]` rather than `2,4-,4,4+,2,7 [4+ A1],3,3,3`. 

I think it is reasonable to assume you could also use the later format. 

I have only fixed this for new routes. Existing data is NOT fixed. 

If you come across this in an existing route you have to first change gear style then save then change gear style back and save again. This forces the system to re-assess the overall grade.

**Comment by killakalle** [comment]
@scd 

Is this hotfixed or only expected to be within the next release?

After I saw your comment I've played around with this myself (including changing gear style), but couldn't actually see any difference, i.e. aid grade not displayed other than in the grade reference.

https://www.thecrag.com/es/escalar/spain/montanejos/barranco-de-la-maimona/route/2340464775
![image](https://user-images.githubusercontent.com/30682796/168896213-9417821d-1707-4453-a795-ef656ca1656f.png)

1. Is this already deployed and should be effective?
2. Is this route a good example for the fix you provide, should the aid grade be displayed anywhere else than the grade reference.
